canonical-web-and-design/feedinghopeandheroes.org
# Move files into correct location and add google form backends'

## Done

- Moved and renamed files to work with nginx
- Added google forms to backend the forms

## QA

- Download this branch
- run the dotrun snap
- See the site runs and the top two forms sumbit.
